README.md

```diff
@@ -4,32 +4,68 @@ Export license usage for SideFX license server to Prometheus.
 
 ## Requirements
 
-To run this exporter, you need to provide the binaries `hserver` and `sesictrl`.
-They are part of the houdini install on Linux, and you can retrieve them from
-their respective destination paths after installing houdini :
+This project is expected to run in a Docker container, so you'll need a Docker
+engine.
+
+### Binaries
+
+To run this exporter, you need the `hserver` and `sesictrl` binaries.
+They are part of the houdini install on Linux, and can be found here :
 
 - `/opt/hfs<version>/bin/hserver`
 - `/opt/hfs<version>/houdini/sbin/sesictrl`
 
-The version of houdini (which translates 1:1 with the version of these binaries)
-should be above the version of your license server. At the time of writing, the
-exporter has been tested with versions 19.5 and 20.
+Make sure your Houdini version matches or is newer than your license server
+version. Currently, this exporter has been tested with versions 19.5 and 20.
+
+The available Docker compose configuration expects these binaries to be
+installed in the `./hfs-bin/` directory, but you can use the directory of your
```

Comment:
Can you precise the way to choose the directory?

Reply:
There might be a bit of complexity between the docker-compose and the `config.yml` file that will be interpreted from inside the container that could be worth explaining :+1:

Reply:
I fixed that in 84eb035, hopefully this is better.